local/sum.js

```diff
@@ -0,0 +1,7 @@
+
+function sum(a, b) {
```

Comment:
Id suggest adding the 'local' directory to your .gitignore so it doesnt get pushed up to github